Cakes-Co/index.html

```diff
@@ -7,10 +7,108 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+     <link rel="stylesheet" href="style.css">
+     <link href="https://fonts.googleapis.com/css2?family=Pacifico&family=Poppins:wght@400;700&display=swap" rel="stylesheet">
 </head>
-
 <body>
     <!-- Add your markup here -->
+
+
+
+
+   
+<header class="header">
+    <div class="gray-cover"></div>
+    <h1 class="header-text"> The best cakes in the town <br>delivered to your door</h1>
+</header>
+
+
+
+<nav class="navbar">
+    <ul class="navlist">
+        <li> <a href="#home">Home</a></li>
+        <li> <a href="#cakes">Cakes</a></li>
+        <li> <a href="#order">Order</a></li>
+        <li> <a href="#about">About</a></li>
+    </ul>
+</nav>
+
+
+
+
+<main class="main-container">
+
+
+    <section class="text-section">
+        <h2 class="main-text">Welcome to our world of cakes!</h2>
+        <br>
+        <p>
+Discover a tempting selection of freshly baked cakes, made with love and the finest ingredients. Whether you're celebrating a special occasion or simply indulging, we have the perfect cake to make every moment sweeter. Explore our range and find your next delicious treat!</p>
+    </section>
+
+    
+    <div class="crop-img">
+
+      <aside"> <img src="https://images.pexels.com/photos/8226211/pexels-photo-8226211.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="chocolate cake" class="aside-img"> </aside>  
+    </div>
+    
+</main>
+
+
+
+
+
+
+
+
+
+
+<div class="text-container">
```

Comment:
One semantic observation here. Why not include this `div` and the next `section` in the `<main> `? As I see it forms the main content of the page.